pkg/packages.builds

```diff
@@ -3,7 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)\projects\*\*.builds" />
+  	<ProjectExclusions Include="$(MSBuildThisFileDirectory)\projects\*\Microsoft.NETCore.UniversalWindowsPlatform.builds" Condition="'$(OSEnvironment)' != 'Windows_NT'" />
```

Comment:
BTW, we use the same approach in [corefx](https://github.com/dotnet/corefx/blob/master/src/src.builds#L5-L7) for similar situations.